microsoft-edge/progressive-web-apps-chromium/experimental-features/index.md

```diff
@@ -264,7 +262,7 @@ Besides the JavaScript API, you may use CSS to query the bounding rectangle of t
 Users expect to grab and drag the upper region of a window.  To accommodate the expectation, declare specific parts of the web content as draggable.  
 To specify an element is draggable, use the webkit proprietary `-webkit-app-region` CSS property.  The CSS working group continues efforts to standardize the `app-region` property.  
 
-To preview this feature in Microsoft Edge for desktop OSs, navigate to [Turn on experimental features](#turn-on-experimental-features) and navigate to **Desktop PWA Window Controls Overlay**.   
+To preview this feature in Microsoft Edge for Windows 10, navigate to [Turn on experimental features](#turn-on-experimental-features) and navigate to **Desktop PWA Window Controls Overlay**.   
```

Comment:
Can you remove this one? It's under the "draggable regions" section, but the feature flag enables the whole feature, not just this one piece.

Reply:
Or move down just a few lines so its under the "Custom title bar example" header?

Reply:
Moved it up again to before the title bar examples.